src/main/java/zoo/Main.java

```diff
@@ -0,0 +1,61 @@
+/* Create different animals. Show menu to a user and ask the user to choose point of menu.
+For each point of menu show information about animal.
+*/
+
+package zoo;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+
+        int pointOfMenu = 0;
+
+        String[] listOfAnimals = {"Lion", "Carp", "Parrot"};
+
+        Scanner scanner = new Scanner(System.in);
+
+        Mammal Lion = new Mammal("Lion", "roars", "earth", "air");
```

Comment:
Here and below: remember the naming conventions, this is just a variable, the name should be in camelCase